.gitlab-ci.yml

```diff
@@ -93,7 +93,7 @@ install:
 # Build a container image async, and don't block CI tests
 # Cache intermediate images for 1 week (168 hours)
 build-review-image:
-  stage: review
+  stage: build
```

Comment:
I think this should stay in the `review` stage. It should already run at the beginning of the pipeline because of the `needs: []` on the line below.

Reply:
doh, you are right